### ISSUE TITLE
Backup file generator. Title, actual dts, displayed dts, content, tags.

### ISSUE THREAD

**Issue by chulman444** [chulman444/Muniverse#31]
JUST IN CASE!

Also, backup-like file reader to read the file and create a new Muniverse thing out of it. For example:

^[YYYYMMDDhhmm[[-.\~;:]?]YYYYMMDDhhmm // Displayed. Optional start_dt
[YYYYMMDDhhmm[-.\~;:]?YYYYMMDDhhmm] // Optional. Actual open\~end dt ... or start\~saved dt. Both date time things accept 8 or 16 digits, two 8 digits separated by one special character. Special characters are optiona
[Title] // title is optional
// blank space if using title
Content
\[ ... tags go here ... \]$ // Actual square brackets. Or change if regex becomes too complex.

**Comment by chulman444** [comment]
Right now, I have my development file with develop and master branch. Pushing master branch to this repository, as well as storing my own notes on the master.

I also used to download this and took notes there as well? Even though I removed them... I don't know if I took real notes or not.

Anyways, for now, the development AND my master branch use the same storage file. lol. No wonder why I started to well written notes when using develop-branch.

My shit files made within develop-branch are so many that it would take a long time till notes I write in my master branch catch up to the last-insert-id of the develop-branch, but there needs to be backup file at least. I'm starting to migrate from google docs to here.
